src/main/scala/com/databricks/spark/sql/perf/mllib/BenchmarkAlgorithm.scala

```diff
@@ -82,7 +90,9 @@ trait TestFromTraining {
     // Copy the context with a new seed.
     val ctx2 = ctx.params.randomSeed match {
       case Some(x) =>
-        val p = ctx.params.copy(randomSeed = Some(x + 1))
+        // Also set the number of examples to the number of test examples.
```

Comment:
Yes, I tried to do this at first, but it turns out it is useful when building the training set from a 'true' model.